molecule/prerequisites_only/tests/test_prerequisites_only.py

```diff
@@ -0,0 +1,44 @@
+"""Module containing the tests for the prerequisites_only scenario."""
+
+# Standard Python Libraries
+import os
+
+# Third-Party Libraries
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ["MOLECULE_INVENTORY_FILE"]
+).get_hosts("all")
+
+
+@pytest.mark.parametrize("pkg", ["apt-transport-https"])
```

Comment:
See previous comment about `apt-transport-https`.

Reply:
Please see commit 021a522.  I will do a grand cleanup of the commit history once the PR is approved.